<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>312</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeAllCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(65, 145, 225, 0.0); color :rgba(212, 212, 212, 0.93)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeAllCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(65, 145, 225, 0.0); color :rgba(212, 212, 212, 0.93)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeAllCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(65, 145, 225, 0.0); color :rgba(212, 212, 212, 0.93)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeAllCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(65, 145, 225, 0.0); color :rgba(212, 212, 212, 0.93)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_8">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeAllCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(65, 145, 225, 0.0); color :rgba(212, 212, 212, 0.93)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeAllCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(65, 145, 225, 0.0); color :rgba(212, 212, 212, 0.93)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeAllCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(65, 145, 225, 0.0); color :rgba(212, 212, 212, 0.93)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeAllCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(65, 145, 225, 0.0); color :rgba(212, 212, 212, 0.93)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeAllCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(65, 145, 225, 0.0); color :rgba(212, 212, 212, 0.93)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>436</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-style: solid; border-radius: 10px; background-color:rgba(26, 97, 168, 0.55); color:white;}
QPushButton:Hover{  background-color:rgba(26, 146, 168, 0.55); color: black; border: 2px solid rgba(26, 97, 168, 0.99);}</string>
    </property>
    <property name="text">
     <string>Neue Spiel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;background-color:rgba(255, 255, 255, 0.75); border-radius:10% 10% 0% 0%;</string>
    </property>
    <property name="text">
     <string>X wird anfangen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(65, 145, 225, 0.56); border-radius:10px; color : white</string>
    </property>
    <property name="text">
     <string>Gewinner</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>65</y>
      <width>141</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 0.36); border-radius:10px; color: rgba(0, 219, 110, 1); </string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>380</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-style: solid; border-radius: 10px; background-color:rgba(255,255, 255, 0.55); color: black;}
QPushButton:Hover{  background-color: rgba(168, 26, 26, 0.55); color:white; border: 2px solid rgba(255,255, 255, 0.99);}</string>
    </property>
    <property name="text">
     <string>Liste löschen</string>
    </property>
    <property name="iconSize">
     <size>
      <width>21</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;background-color:rgba(255, 255, 255, 0.75); border-radius:10% 10% 0% 0%;</string>
    </property>
    <property name="text">
     <string>Games: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-style: solid; border-radius: 10px; background-color:rgba(26, 97, 168, 0.55); color:white;}
QPushButton:Hover{  background-color:rgba(222, 222, 222, 0.55); color: black; border: 2px solid rgba(26, 97, 168, 0.99);}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../Desktop/imag/source.qrc"/>
 </resources>
 <connections/>
</ui>
